modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -27,96 +32,180 @@ The repository must be accessible from the hub cluster and be defined as a sourc
 
 .Procedure
 
-. Create a `ConfigMap` resource that describes the NICs for a group of hosts. For example:
+. Create three `ConfigMap` resources that describe the group and site configuration. 
++
+.. Create `group-hardware-types-configmap`, a `ConfigMap` to hold the hardware specific configuration. For example:
 +
 [source,yaml]
 ----
+---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: sriovdata
-  namespace: ztp-site
+  name: group-hardware-types-configmap
+  namespace: ztp-group
   annotations:
     argocd.argoproj.io/sync-options: Replace=true <1>
 data:
-  example-sno-du_fh-numVfs: "8"
-  example-sno-du_fh-pf: ens1f0
-  example-sno-du_fh-priority: "10"
-  example-sno-du_fh-vlan: "140"
-  example-sno-du_mh-numVfs: "8"
-  example-sno-du_mh-pf: ens3f0
-  example-sno-du_mh-priority: "10"
-  example-sno-du_mh-vlan: "150"
+  # SriovNetworkNodePolicy.yaml - general
+  dell-poweredge-xr12-sriov-node-policy-pfNames-1: "[\"ens5f0\"]"
```

Comment:
We should probably change all occurrences of "dell-poweredge-xr12" to "hardware-type-1" to make it more generic.